sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClientBuilderTest.java

```diff
@@ -1,30 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.messaging.servicebus.administration;
-
-import com.azure.core.http.policy.HttpLogOptions;
-import com.azure.core.util.ClientOptions;
-import org.junit.jupiter.api.Test;
-
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
-/**
- * Unit tests for {@link ServiceBusAdministrationClientBuilder}.
- */
-public class ServiceBusAdministrationClientBuilderTest {
-
-    @Test
-    void buildClientDifferentApplicationIdTest() {
```

Comment:
Any reason for deleting this file?

Reply:
This was the only test in the file but with the change to use ClientOptions, else HttpLogOptions without throwing the test no longer applied. This left the class without any tests